src/libs/actions/FormActions.ts

```diff
@@ -0,0 +1,26 @@
+import Onyx from 'react-native-onyx';
+import {PartialDeep} from 'type-fest';
+import {KeyValueMapping} from 'react-native-onyx/lib/types';
+import * as OnyxCommon from '../../types/onyx/OnyxCommon';
+import {OnyxKey, OnyxValues} from '../../ONYXKEYS';
+
+type KeysWhichCouldBeDraft<T extends keyof OnyxValues> = T extends `${infer U}Draft` ? U : never;
+type DraftKeysWithoutDraftSuffix = KeysWhichCouldBeDraft<keyof OnyxValues>;
+
+function setIsLoading(formID: OnyxKey, isLoading: boolean) {
```

Comment:
```suggestion
function setIsLoading(formID: OnyxFormKey, isLoading: boolean) {
```

I guess we can restrict to accept only form keys, we could create this type in `ONYXKEYS.ts`

```ts
type OnyxKeysMap = typeof ONYXKEYS;
type OnyxCollectionKey = ValueOf<OnyxKeysMap['COLLECTION']>;
type OnyxKey = DeepValueOf<Omit<OnyxKeysMap, 'COLLECTION'>>;
type OnyxFormKey = ValueOf<OnyxKeysMap['FORMS']>; // new type
```